AppCenter/AppCenter/Internals/Channel/MSOneCollectorChannelDelegate.m

```diff
@@ -4,16 +4,19 @@
 #import "MSCommonSchemaLog.h"
 #import "MSCSEpochAndSeq.h"
 #import "MSOneCollectorChannelDelegatePrivate.h"
+#import "MSOneCollectorIngestion.h"
 #import "MSUtility.h"
 
 static NSString *const kMSOneCollectorGroupIdSuffix = @"/one";
+static NSString *const kMSOneCollectorBaseUrl = @"https://browser.events.data.microsoft.com"; // TODO: move to constants?
```

Comment:
This url is reserved for web client. 
Please use "https://mobile.events.data.microsoft.com" as the One Collector endpoint for mobile.